db/migrate/20241121122712_add_is_confidential_to_references.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsConfidentialToReferences < ActiveRecord::Migration[8.0]
+  def change
+    add_column :references, :is_confidential, :boolean, default: true, null: false
```

Comment:
How do you feel about renaming this to `confidential` dropping the `is_`?

Reply:
Fair challenge, `confidential` is nicer. I've updated.